python/tests/complex_type_tests/test_row.py

```diff
@@ -0,0 +1,32 @@
+from tests.aggregate_tests.aggtst_base import TstView, TstTable
+
+
+# Depth 1 tests
+class cmpxtst_row_int_var_tbl(TstTable):
+    """Define the table used by the ROW of int and varchar tests"""
+
+    def __init__(self):
+        self.sql = """CREATE TABLE row_int_var_tbl(
+                      id INT,
+                      c1 ROW(i1 INT NOT NULL, i2 INT NULL),
```

Comment:
for ROW you can specify nullability, so please make `c1 ROW(...) NOT NULL` and ad c3 ROW(...) NULL`.